css/src/components/form/input.scss

```diff
@@ -1,10 +1,11 @@
 $input-color: $text !default;
```

Comment:
Validation state borders should be updated as well. On focus validation color should be reset:

![image](https://github.com/user-attachments/assets/7f87ab3c-3cc3-456a-8414-9423b7e8bcc2)

focus

![image](https://github.com/user-attachments/assets/059677e1-8ef6-417c-92e3-86dbe8e5783e)


https://react.fluentui.dev/?path=/docs/components-field--default#validation-message

Reply:
On all controllers*

Reply:
Isn't that more of an issue for form-behavior?

Reply:
@ellynliu  please revert this back to solid red color. I found the [Fluent 2 Figma file](https://www.figma.com/design/By5idlaqQdmhTTnabMkuKk/Microsoft-Fluent-2-Web-(Community)?node-id=8934-4&t=f5KPvAQzXYLSgtJI-0), which is the main source we should rely on, and for the error state border is red at all times.

![image](https://github.com/user-attachments/assets/4b0772e6-fb3d-4723-85b8-a97549daead9)